desktop.css

```diff
@@ -0,0 +1,543 @@
+  font-family: "Poppins", sans-serif;
+  color: #091e42;
+}
+
+header a {
+  text-decoration: none;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 16px;
+  border-radius: 1em;
+  padding-bottom: 12px;
+  background-color: #fff;
+  margin-bottom: 20px;
+  position: fixed;
+}
+
+#logo {
+  font-weight: bold;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  margin-left: 143px;
+}
+
+.nav-items ul a {
+  color: black;
+}
+
+#logo a {
+  color: #6070ff;
+}
+
+.nav-items ul {
+  list-style: none;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-right: 142px;
+}
+
+.nav-items li {
+  padding: 8px;
+  margin-left: 10px;
+}
+
+.bar {
+  display: none;
+  top: 50px;
+  left: 0;
+  width: 100%;
+  margin-top: 10px;
+}
+
+.bar div {
+  width: 35px;
+  height: 3px;
+  background-color: black;
+  margin: 5px 0 5px 280px;
+  transition: 0.4s;
+}
+/* stylelint-disable */
+.container {
+  border-radius: 1em;
+  position: static;
+  padding-top: 200px;
+  padding-left: 260px;
+  padding-right: 260px;
+  background-color: white;
+  padding-bottom: 280px;
+  background-image: url(./images/Ellipse-lg3.svg), url(./images/Ellipse-11.png),
+    url(./images/Ellipse-4.svg), url(./images/Rectangle-4.svg);
+  background-repeat: no-repeat;
+  background-position: right 150px, 139.2px 60.68px, 839px 608px, 1150px 120px;
+}
+/* stylelint-enable */
+.container .lp {
+  display: none;
+}
+
+.header-text {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 48px;
+  line-height: 60px;
+}
+
+.text {
+  color: #344563;
+  font-family: Poppins, sans-serif;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  margin-top: 12px;
+}
+
+.connect {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.projects {
+  display: grid;
+  border-radius: 1rem;
+  padding-top: 100px;
+  margin-bottom: 114px;
+  margin-top: 12px;
+}
+
+.socialmedia {
+  float: left;
+  display: flex;
+  list-style-type: none;
+}
+
+.socialmedia li {
+  padding-right: 20px;
+}
+
+.card {
+  display: flex;
+  flex-direction: row;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  margin-left: 140px;
+  margin-right: 140px;
+  margin-top: 88px;
+  padding-bottom: 20px;
+}
+
+.card-rev {
+  display: flex;
+  flex-direction: row-reverse;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  margin-left: 140px;
+  margin-right: 140px;
+  margin-top: 88px;
+  padding-bottom: 20px;
+}
+
+.one {
+  width: 50%;
+}
+
+.two {
+  width: 50%;
+  margin-left: 50px;
+  padding-top: 48px;
+}
+
+.card img {
+  left: 16px;
+  top: 16px;
+  padding: 16px;
+  align-items: center;
+  width: 100%;
+}
+
+.card-rev img {
+  left: 16px;
+  top: 16px;
+  padding: 16px;
+  align-items: center;
+  width: 100%;
+}
```

Comment:
- [ ] Kindly your image overflows the design on the desktop version. Please fix its width to be like the design.
![image](https://user-images.githubusercontent.com/35535130/154477698-274453b2-86b6-4388-89e7-a3761b8d9a6b.png)

and here 
![image](https://user-images.githubusercontent.com/35535130/154477774-5caa74b7-9cab-4388-b39d-cf0afe5e812e.png)